### PR TITLE
Web Lit Basics II

### LINKED ISSUE #1433: Create Web Lit Basics II teaching kit
### Part 1: Create Page ###

url: /activities/web-lit-basics-two

Page content is [here](https://docs.google.com/document/d/1TK2AAc5Bhe0sO1plj_ZtfI7LIMoc7rHnzmHoTt_URMI/edit#)

Note: This Teaching Kit does note follow the "Reading the Web," "Writing the Web," "Participating on the Web" structure of our other kits, so those header styles won't be needed. Instead, we can just list the activities directly under the Learning Objectives.

### Part 2: Add to /activities ###

@chadsansing - can you please fill in the below?

Web Literacy Basics II (Teaching Kit)
level: 
developed by: 
description:
image: https://c2.staticflickr.com/6/5760/22431506387_43e85e71bd.jpg

### PR DESCRIPTION
**Pull request by mmmavis**
Fixes #1433 

notes/todos:

- [x] fix a broken link (link to "The Web is a Tool for Learning")

### CODE REVIEW

File: pages/web-lit-basics-two.jsx [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+          description: "Learners will create and research survey questions about their community's Web use, learning about <strong>collaborating, community participation, and open practices</strong>.",
+          link: "https://chadsansing.github.io/curriculum-testing/web-lit-basics-two/session01-why-do-we-use-the-web.html"
+        },
+        {
+          title: "The Web is a Tool for Learning",
+          image1x: "/img/pages/web-lit-basics-two/the-web-is-a-tool-for-learning.jpg",
+          originalImgSrc: "https://c1.staticflickr.com/1/88/234010602_0b42343b8d.jpg",
+          subtitle: "Understanding Navigating",
+          description: "Learners will use Web-native instructions to make a meme or build a maker project in real life, learning <strong>navigating, remixing, and community participation</strong>.",
+          link: "https://chadsansing.github.io/curriculum-testing/web-lit-basics-two/session02-the-web-is-for-making.html"
```
Comment (Pomax) [inline pages/web-lit-basics-two.jsx]: there's a whole bunch of `http` links for chad's github pages, can we change all of those to `https`?

File: pages/web-lit-basics-two.jsx [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+  ],
+  render: function () {
+    var blogPostLink = "https://blog.webmaker.org/help-us-get-local-with-web-literacy";
+    return (
+      <div className="inner-container">
+        <CurriculumIntro/>
+        <WebLitBasics/>
+        {this.curriculum.map(function (section, key) {
+          return (
+            <ActivitySection title={section.title} key={key} activities={section.activities} />
```
Comment (Pomax) [inline pages/web-lit-basics-two.jsx]: keys are for identifying items, so we can't use the array position. The title is a good unique identifier, though, so we can use `key={section.title}`

**Comment by Pomax** [comment]
pretty close to r+

**Comment by mmmavis** [comment]
@Pomax I updated my PR with fixes! Can you quickly review it again for me? :+1: 

I'm not sure if I credit the images correctly though - most of them link to Chad's Github pages but I'm sure those aren't our original creations either :grimacing:

**Comment by hannahkane** [comment]
@chadsansing - please advise re: mavis's comment above about crediting images

**Comment by chadsansing** [comment]
@hannahkane, @mmmavis: ship me the URL(s) for image-crediting review? Mondo thanks!

**Comment by mmmavis** [comment]
@chadsansing 
screen shot 2015-12-02 at 11 29 14 am
the "view original" links(see screencap) link to the img src provided in this [doc](https://docs.google.com/document/d/1TK2AAc5Bhe0sO1plj_ZtfI7LIMoc7rHnzmHoTt_URMI/edit#heading=h.jrd2718o4kpj) e.g., http://chadsansing.github.io/curriculum-testing/web-lit-basics-two/space-cat.png

**Comment by chadsansing** [comment]
Sorry to be obtuse, @mmmavis  - I hope this is the info you need :) Links can also be found in the a element surrounding the credit/i element in each page's sidebar (around line 55 or so).

- **bridge01** - CC-BY MLN, links to teach dot
- **bridge02** - CC-BY MLN, links to teach dot
- **bridge03** - pubic domain intangible idea, links to CC0 1.0 public domain by author neutrality, links to https://commons.wikimedia.org/wiki/File:Question_mark_%28black_on_white%29.png
- **session01** - CC0 1.0 public domain by author Tungalik, links to https://commons.wikimedia.org/wiki/File:Online_Survey_Icon_or_logo.svg
- **session02** - CC-BY 2.0 by rahims, links to https://www.flickr.com/photos/rahims/234010602
- **session03** - CC-BY-SA 2.0 by badjonni, links to https://www.flickr.com/photos/badjonni/433463323/sizes/n/
- **session04** - bridge02 - CC-BY MLN, links to teach dot
- **session05** - public domain common knowledge, by Zscout370, Sertion, e.a., links to https://en.wikipedia.org/wiki/Copyleft#/media/File:Copyleft.svg
- **session06** - CC-BY-SA 3.0 by R. Jay GaBany, links to https://commons.wikimedia.org/wiki/File:NGC_3521-_the_Bubble_galaxy.jpg

**Comment by mmmavis** [comment]
Thanks @chadsansing ! This is very helpful!

@Pomax I updated the PR with proper image credits!

**Comment by chadsansing** [comment]
Plusses, @mmmavis.

**Comment by mmmavis** [comment]
I think this PR is ready to go.  @Pomax if you find any bugs when you are back tomorrow just give me a ping and I'll submit a patch for it.

Merging this now.

**Pull request merged**